editor/src/app/sites/settings/site-settings.state.ts

```diff
@@ -55,4 +55,22 @@ export class SiteSettingsState implements NgxsOnInit {
       [action.settingGroup]: updatedSiteSettingsGroup
     }});
   }
+
+  @Action(RenameSiteSettingsSitenameAction)
+  renameSiteSettingsSitename({ setState, getState }: StateContext<SitesSettingsStateModel>, action: RenameSiteSettingsSitenameAction) {
+    const state = getState();
+    const newState = cloneDeep(state);
```

Comment:
Do not use `cloneDeep` for no reason - you're only updating part of the state here. This will cause the whole state object and all its sub-objects to be changed and everywhere where it's used there will be re-renders.

You should use `Object.assign`, spread `{...}` or `for ... in` loop here.

See my PR: [Simple setting store updates](https://github.com/berta-cms/berta/pull/144/files#diff-5d8bfe1b10f39b05bbf2bde0474ab3b4) for deep updates.